docs/benchmark.md

```diff
@@ -0,0 +1,72 @@
+# Benchmark
+
+Go benchmark tests that measure end-to-end performance of a running Ollama server. Run these tests to evaluate model inference performance on your hardware and measure the impact of code changes.
```

Comment:
```suggestion
Go benchmark tests that measure end-to-end performance of a running Ollama server. Run these tests to evaluate model inference performance on your hardware and measure the impact of code changes
```

Reply:
Are there other places we don't use a period? The intro to the main README has one

Reply:
IIRC we didn't use periods but maybe I'm wrong?

Reply:
The other intros in the docs folders seem to be using periods, so gonna leave this one for now.